CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.16.0 Release
+- Release Date: February 16, 2019
+- Server Versions Supported: Server v4.10+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Combatibility
+
+ - Mobile App v1.13+ is required for Mattermost Server v5.4+.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+
+### Improvements
+ - Added the ability to remove own profile picture.
+ - Changed "X" to "Cancel" on Edit Profile page.
+ - Added support for relative permalinks.
+
+### Bug Fixes
+ - Fixed an issue where the iOS app did not trigger MobileIron VPN to connect.
```

Comment:
@enahum what about the russian translation issue? Even if the code change wasn't on this repo I think we should mention it

Reply:
Looks good otherwise

Reply:
Yes russian should be mentioned.

Also the VPN connection should be: `Fixed an issue where the iOS app did not waited until the on-demand VPN connection was established. (EMM Providers)` this is not a MobileIron specific case